managed/services/inventory/grpc/services_server.go

```diff
@@ -88,6 +88,25 @@ func (s *servicesServer) ListServices(ctx context.Context, req *inventorypb.List
 	return res, nil
 }
 
+// ListActiveServiceTypes returns list of active Services.
+func (s *servicesServer) ListActiveServiceTypes(
+	ctx context.Context,
+	req *inventorypb.ListActiveServiceTypesRequest,
+) (*inventorypb.ListActiveServiceTypesResponse, error) {
+	types, err := s.s.ListActiveServiceTypes(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	res := &inventorypb.ListActiveServiceTypesResponse{
+		ServiceTypes: make([]inventorypb.ServiceType, 0, len(types)),
+	}
+
+	res.ServiceTypes = append(res.ServiceTypes, types...)
```

Comment:
It seems we can just do it like this:
```suggestion
	res := &inventorypb.ListActiveServiceTypesResponse{
		ServiceTypes: types,
	}
```

Reply:
Great catch